pkg/proxy/util/conntrack.go

```diff
@@ -56,3 +57,34 @@ func ExecConntrackTool(execer exec.Interface, parameters ...string) error {
 	}
 	return nil
 }
+
+// ClearUDPConntrackForPort uses the conntrack tool to delete the conntrack entries
+// for the UDP connections specified by the port.
+// When a packet arrives, it will not go through NAT table again, because it is not "the first" packet.
+// The solution is clearing the conntrack. Known issues:
+// https://github.com/docker/docker/issues/8795
+// https://github.com/kubernetes/kubernetes/issues/31983
+func ClearUDPConntrackForPort(execer exec.Interface, port int) {
```

Comment:
Yes, it should return the error.

BTW, I find the existing DeleteServiceConnections() doesn't return the error, see

https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/util/conntrack.go#L34-L45

Should I fix them together?